### DISCUSSION TITLE
Error when importing pyreadstat in Jupyter notebook

### DISCUSSION THREAD

**Discussion by erikhhs** [Roche/pyreadstat#169] [Q&A] [answered]
Dear,

When I try to import pyreadstat into a Jupyter notebook I get an error:

AttributeError: module 'pandas' has no attribute 'Float64Dtype'.  

The latest version of pandas has been installed. I hope you can help me with this?

Kind regards,

Erik

**Comment by ofajardo** [comment, marked as answer]
what do you get if you do: 

```
import pandas as pd
print(pd.__version__)
```

**Reply by erikhhs** [reply]
1.1.3

**Reply by ofajardo** [reply]
current pandas version is 1.4.1. I think you need at least pandas 1.2.1 for pyreadstat to work correctly. If for some reason you cannot update to that, you will need to downgrade the pyreadstat version.

**Reply by erikhhs** [reply]
Thanks Otto, updating to 1.4.1 solved the problem!

**Reply by ofajardo** [reply]
happy to have helped!
